src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -151,6 +152,7 @@ CATCH_RETURN()
 
     return S_OK;
 }
+CATCH_RETURN()

```

Comment:
I don't know why auditing is complaining about possible exceptions even though I clearly added `CATCH_RETURN()`.

Reply:
I remember having this problem before, but not what I did about it besides rip off the `noexcept` and go on with life.